### ISSUE TITLE
Wrong command for local init

### ISSUE THREAD

**Issue by Thomas0c** [holidaypirates/nucleus#1]
NPM version: 3.10.3
Node: 6.3.0
Release: 1.0.3

Documentation (https://holidaypirates.github.io/nucleus/installation.html) states that per-project installation initialization can be run using command `node ./node_modules/bin/nucleus`.

Very same command throws an error, this seems to be due to the fact that the module is installed as `nucleus-styleguide`, and that the module is added to `node_modules` and not `node_modules/bin`.

Please see error below:
```
module.js:442
    throw err;
    ^

Error: Cannot find module '/Users/username/Sites/website/node_modules/bin/nucleus'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
```
The command that worked for me was therefore: `node ./node_modules/nucleus-styleguide`.

**Comment by michaelseibt** [comment]
Hi and thanks for pointing this out. I will verify and fix this as soon as possible.

**Comment by Thomas0c** [comment]
Updated w. error message, if that is of any help.
Good job on the project btw - looks very promising.

**Comment by michaelseibt** [comment]
Thanks a lot. Init script is fixed in 1.0.4 and working as expected when calling ```nucleus init```.
